.github/workflows/build.yml

```diff
@@ -81,7 +81,7 @@ jobs:
   test-examples:
     strategy:
       matrix:
-        node: [14, 16, 18]
+        node: [16, 18]
```

Comment:
Are we not planning to support 14 anymore? I know it's EOL and I'm open to the convo, just don't want it to be an arbitrary decision.

Reply:
@cprice404 left comment here 
https://github.com/momentohq/client-sdk-javascript/pull/643/files#r1265604485

Dont have super strong opinions on this. I think would be nice to not support 14 since its EOL but could also break this out to a `3rdParty-examples` project where we run on smaller subsets of node for examples. Would need to update docs as well then to pull dynamically from there also.